.gitignore

```diff
@@ -1,48 +1,19 @@
-# Logs
```

Comment:
I think it was nice to leave the extension `log` files.

Reply:
We don't have any log on our module!